next.config.js

```diff
@@ -1,4 +1,16 @@
 /** @type {import('next').NextConfig} */
 module.exports = {
   reactStrictMode: true,
+  images: {
+    remotePatterns: [
+      {
+        protocol: 'https',
+        hostname: '**',
+      },
+      {
+        protocol: 'http',
+        hostname: '**',
```

Comment:
This is also unrelated but necessary for arbitrary URLs, you want me to submit a separate PR for that?